src/acceptance-test/java/de/zalando/aruha/nakadi/webservice/utils/KafkaHelper.java

```diff
@@ -0,0 +1,74 @@
+package de.zalando.aruha.nakadi.webservice.utils;
+
+import de.zalando.aruha.nakadi.domain.Cursor;
+import org.apache.kafka.clients.consumer.KafkaConsumer;
+import org.apache.kafka.clients.producer.KafkaProducer;
+import org.apache.kafka.clients.producer.ProducerRecord;
+import org.apache.kafka.common.TopicPartition;
+
+import java.util.List;
+import java.util.Properties;
+import java.util.concurrent.ExecutionException;
+import java.util.stream.Collectors;
+
+public class KafkaHelper {
```

Comment:
Agree. I now renamed it within another PR that is based on this branch: https://github.com/zalando/nakadi/pull/84